mappings/net/minecraft/world/biome/source/MultiNoiseBiomeSourceParameterList.mapping

```diff
@@ -1,12 +1,14 @@
-CLASS net/minecraft/class_8197 net/minecraft/world/biome/source/MultiNoisePreset
-	FIELD field_42986 PRESET_CODEC Lcom/mojang/serialization/Codec;
+CLASS net/minecraft/class_8197 net/minecraft/world/biome/source/MultiNoiseBiomeSourceParameterList
```

Comment:
This is the name given by the registry, and the directory where these are stored. They *contain* what is called a Preset (as a parameter), and aren't themselves Presets. They will likely have more parameters added to them later.

Reply:
They _are_ themselves presets - the `PRESET_CODEC` in `MultiNoiseBiomeSource` that can be used instead of directly passing the biomes now uses the `REGISTRY_CODEC` of this class.